src/pages/ExamsPage/hooks.js

```diff
@@ -57,6 +57,11 @@ export const useInitializeExamsPage = (courseId) => {
   React.useEffect(() => { fetchCourseExams(courseId); }, []); // eslint-disable-line react-hooks/exhaustive-deps
 };
 
+export const useUpdateCurrentExamIndex = () => {
+  const dispatch = useDispatch();
+  return (examId) => () => dispatch(reducer.setCurrentExamIndex(examId));
```

Comment:
Because there is no network request required here (since this isn't changing exams), I wasn't sure if this was the correct way to update the state of the current exam index.

Reply:
seems right to me, I'll probably double check against some other frontend code to be sure tho